docs/en/operation/Metrics-System.md

```diff
@@ -126,3 +126,84 @@ to the under store.
 
 Tags are additional pieces of metadata for the metric such as user name or under storage location.
 Tags can be used to further filter or aggregate on various characteristics.
+
+## Setting up Grafana for Alluxio
+
+Grafana is a metics analytics and visualization software used for visualizing time series
+data. You can use Grafana to better visualize the various metrics that Alluxio collects. The software
+allows users to more easily see what's going on behind the scenes in Alluxio.
+
+Before configuring Grafana a monitoring tool must be set up that Grafana will pull metrics from. Alluxio
```

Comment:
The part "..tool must set up that Grafana will pull metrics from" sounds weird to me. The grammar is a little off I think. Can we try to reword?